tests/phpunit/class-wc-stripe-webhook-handler-test.php

```
@@ -339,6 +339,111 @@ function ( $passed_order ) use ( $order ) {
+							&& isset( $passed->data->object->id )
+							&& $notification->data->object->id === $passed->data->object->id;
+					}
+				)
+			);
+
+		$handler->process_deferred_webhook(
+			$event_type,
+			[ 'session_id' => $checkout_session_id ],
+			$notification
+		);
+	}
```

PHPMD (2.15.0)

[error] The method test_process_deferred_webhook_invokes_handle_checkout_session_success is not named in camelCase. (undefined)

(CamelCaseMethodName)